docs/sections/cli_features.md

```diff
@@ -54,6 +54,36 @@ or to execute all the workflows in a project:
 popper run --recursive
 ```
 
+## Specifying extra engine configuration
+
+Many times, workflows might require the containers with some special configuration
+like extra privileges, extra volume bindings, extra capabilites, etc i-e users
+might want to use the different runtime features that container technologies provide.
+Since, Popper by default spawns containers in their most basic and default configuration,
+a `settings.py` file can be passed to `--engine-conf` option of `popper run`
+to specify the extra configuration that is required for the particular use case.
```

Comment:
some tweaks:

By default, Popper instantiates containers in the underlying engine by using basic configuration options (see [here](https://popper.readthedocs.io/en/latest/sections/gha_workflows.html?highlight=mount#execution-runtime)). When these options are not suitable to your needs, you can modify or extend them by providing engine-specific options. These options allow you to specify fine-grained capabilities, bind-mounting additional folders, etc. In order to do this, you can provide a configuration file to modify the underlying container engine configuration used to spawn containers. This file is a python script that defines an `engine_configuration` dictionary with custom options and is passed to the `popper run` command via the `--engine-conf` flag.